tests/testutils/collector_process_test.go

```diff
@@ -134,3 +136,48 @@ func TestCollectorProcessWithInvalidPaths(t *testing.T) {
 	err = collector.Shutdown()
 	require.NoError(t, err)
 }
+
+func TestCollectorProcessWithMultipleConfigs(t *testing.T) {
```

Comment:
These tests are for the actual testutils helper functionality. Mind moving to a new ~`tests/general/process_test.go`? It may make sense to add a new `tc.SkipIfNotProcess()` check: https://github.com/signalfx/splunk-otel-collector/blob/ed553cbcaecc3f3fa86884b1e0a986a9e8ba8518/tests/testutils/testcase.go#L70

Reply:
I'll move the test, thanks for the pointer!

I actually don't think it would make sense to create a `SkipIfNotProcess` check here. `SkipIfNotContainer` is required to make sure the environment is properly setup with a config path to use by default. However, for the `CollectorProcess` utility, the collector default path is set automatically internally, so no default is required.  Let me know if I'm missing something though, happy to add one if there's some extra functionality you think is necessary.

Reply:
`SkipIfNotContainer` checks to see if `SPLUNK_OTEL_COLLECTOR_IMAGE` is set, which is the mechanism used to run the integration tests against the target image as opposed to a built executable, and doesn't really have to do w/ the collector configuration: https://github.com/signalfx/splunk-otel-collector/tree/main/tests/testutils#testcase

https://github.com/signalfx/splunk-otel-collector/blob/main/tests/testutils/testcase.go#L119

By not skipping the added test if the env var is set (which could later be turned into the requested helper) this makes a built executable required for the integration tests to pass which isn't necessarily a requirement otherwise.

Reply:
To make sure I'm understanding properly, you're saying it would be useful to add this check to ensure that the executable is already built successfully for the process tests to run, right?

Reply:
This is something that we can address later as the divergence between test suites grows.